### PR TITLE
Fix skopeo-login docs typo

### PR DESCRIPTION
**Pull request by jarda-wien**

### CODE REVIEW

**Comment by jarda-wien** [comment]
PR is pretty self explanatory. I've just stumbled upon this issue while reading up on the docs.

File: docs/skopeo-login.1.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 skopeo\-login - Login to a container registry
 
 ## SYNOPSIS
-**skoepo login** [*options*] *registry*
+**skopeo login** [*options*] *registry*
```
Comment (TomSweeneyRedHat) [inline docs/skopeo-login.1.md:7]: whoops!

**Comment by TomSweeneyRedHat** [comment]
LGTM
TYVM for the eagle eye and touch up @jarda-wien !

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
Thanks @jarda-wien

**Pull request merged**